### DISCUSSION TITLE
No grid_movement_structure item in the latest version "Common" folder

### DISCUSSION THREAD

**Discussion by HANSIeltsKing** [su2code/SU2#1677] [Q&A]
Hi.

I am unsteady simulation of an airfoil with a flap having pitching motions. Instead of the default sinusoidal motion, I want to defined the motion in the 'grid_movement_structure.cpp' file but it is not in the "Common" folder.

When I open the src folder, it only has a CGridMovement.cpp in the "grid_movement", but this file has nothing about the 'grid_movement_structure.cpp', which is shown below:

```
#include "../../include/grid_movement/CGridMovement.hpp"

CGridMovement::CGridMovement(void) { }

CGridMovement::~CGridMovement(void) { }
```

I would like to know where is the 'grid_movement_structure.cpp' file or relavant items in the latest version?

Thank you very much!

**Comment by pcarruscag** [comment]
Please don't open bug reports for questions. Use the github discussions or join our slack channel. We also have developers meeting every week which are ideal for these questions :) The next one is tomorrow at 10am Pacific time.
https://meet.jit.si/SU2_DevMeeting

**Comment by pcarruscag** [comment]
The code for rigid pitching is CVolumetricMovement::Rigid_Pitching in CVolumetricMovement.cpp
